src/libs/cluster-utils.js

```diff
@@ -83,6 +83,11 @@ export const trimAppsOldestFirst = _.flow(
   _.remove({ status: 'DELETING' }),
   _.sortBy('auditInfo.createdDate'))
 
+// TODO: factor status into cost
+export const hourlyAppCost = (kubernetesRuntimeConfig, status) => {
```

Comment:
If we need multiple pieces of info, we could just pass the whole `app` object. That would simplify the code at the call sites too.